ui/app/scripts/task/controllers/definitions.js

```diff
@@ -90,6 +118,31 @@ define(['model/pageable'], function (Pageable) {
       $scope.bulkDefineTasks = function() {
         $state.go('home.tasks.bulkDefineTasks');
       };
+      $scope.isComposedTask = function(item) {
+        return item && angular.isString(item.dslText) && item.dslText.startsWith('composed-task-runner ');
```

Comment:
May be we can look for `--graph` and probably some syntax that is applicable only for the composed task?